### DISCUSSION TITLE
Feature to merge two images

### DISCUSSION THREAD

**Discussion by gitexplr** [dlemstra/Magick.NET#726] [Help] [answered]
Is there any feature to combine two images based on center point?

For example, I have a blank (white background) image and need to place another image on top of blank image. Output image size would always be size of blank image. 
If the size of first image is bigger than second, final image would have blank (white) outer space. 
If first image size is less than second image size then still the final size would be of first image size (second image would be chopped according to size of first).

**Comment by dlemstra** [comment, marked as answer]
You can use the composite method for that:

```C#
using (var background = new MagickImage(MagickColors.White, width, height))
{
    using (var image = new MagickImage("image"))
    {
        background.Composite(image, Gravity.Center);
    }

    background.Write("output");
}
```

**Reply by gitexplr** [reply]
Thank you for the quick response, saw that option and tried. It worked fine.
